src/include/migraphx/op/pad_calc.hpp

```diff
@@ -0,0 +1,19 @@
+#ifndef MIGRAPHX_GUARD_OPERATORS_PAD_CALC_HPP
+#define MIGRAPHX_GUARD_OPERATORS_PAD_CALC_HPP
+
+#include <utility>
+
+namespace migraphx {
+inline namespace MIGRAPHX_INLINE_NS {
+namespace op {
+
+inline size_t calculate_padding(size_t weight_dim, size_t dilation)
+{
+    return (dilation * (weight_dim - 1)) / 2;
+}
```

Comment:
Can we move this header out of the op directory since it is not an op?

Reply:
fixed